admin_manual/configuration_server/antivirus_configuration.rst

```diff
@@ -89,57 +138,131 @@ Everything.
 
-* Executable: ClamAV is running on the same server as ownCloud, and the
-  ``clamscan`` command is started and then stopped with each file upload.
-  ``clamscan`` is slow and not always reliable for on-demand usage; it is
-  better to use one of the daemon modes.
+**Executable** 
+
+In this mode, ClamAV runs on the same server as the ownCloud installation, and
+the ``clamscan`` command only runs when a file is uploaded. ``clamscan`` is slow
+and not always reliable for on-demand usage; it is better to use one of the
+daemon modes.
 
 Daemon (Socket)
-  ownCloud should detect your ``clamd`` socket and fill in the ``Socket``
-  field. This is the ``LocalSocket`` option in ``clamd.conf``. You can
-  run ``netstat`` to verify::
+^^^^^^^^^^^^^^^
+
+ownCloud should detect your ``clamd`` socket and fill in the ``Socket``
+field. This is the ``LocalSocket`` option in ``clamd.conf``. You can
+run ``netstat`` to verify::
 
    netstat -a|grep clam
    unix 2 [ ACC ] STREAM LISTENING 15857 /var/run/clamav/clamd.ctl
 
   .. figure:: ../images/antivirus-daemon-socket.png
 
-  The ``Stream Length`` value sets the number of bytes read in one pass.
-  10485760 bytes, or ten megabytes, is the default. This value should be 
-  no larger than the PHP ``memory_limit`` settings, or physical memory if 
-  ``memory_limit`` is set to -1 (no limit).
+The ``Stream Length`` value sets the number of bytes read in one pass.
+10485760 bytes, or ten megabytes, is the default. This value should be 
+no larger than the PHP ``memory_limit`` settings, or physical memory if 
+``memory_limit`` is set to -1 (no limit).
 
-  ``Action for infected files found while scanning`` gives you the choice of
-  logging any alerts without deleting the files, or immediately deleting
-  infected files.
+``Action for infected files found while scanning`` gives you the choice of
+logging any alerts without deleting the files, or immediately deleting
+infected files.
 
 Daemon
-  For the Daemon option you need the hostname or IP address of the remote
-  server running ClamAV, and the server's port number.
+^^^^^^
+
+For the Daemon option, you need the hostname or IP address of the remote
+server running ClamAV and the server's port number.
 
   .. figure:: ../images/antivirus-daemon-socket.png
 
 Executable
-  The Executable option requires the path to ``clamscan``, which is the
-  interactive ClamAV scanning command. ownCloud should find it automatically.
+^^^^^^^^^^
+
+The Executable option requires the path to ``clamscan``, which is the
+interactive ClamAV scanning command. ownCloud should find it automatically.
 
   .. figure:: ../images/antivirus-executable.png
 
 When you are satisfied with how ClamAV is operating, you might want to go
 back and change all of your logging to less verbose levels.
 
+Rule Configuration
+^^^^^^^^^^^^^^^^^^
+
+ownCloud provides the abilility to customise how ownCloud reacts to the
+response provided by a ClamAV scan. To do so, under `Admin -> Antivirus
```

Comment:
replace "ClamAV scan." with "Antivirus scan" - The custom rules make only sense, when using other than clamAV ...